### PR TITLE
Update DecompiledRetailConsoleGames.md

### PR DESCRIPTION
**Pull request by FRDS**

### CODE REVIEW

File: pages/SourceCode/RetailConsoleSourceCode.md [hunk 1]
```diff
@@ -37,9 +37,6 @@ Nintendo systems have by far the largest selection of official source code to ch
 ### Days of Thunder 
 {% include link-to-other-site.html url="https://github.com/DickBlackshack/Days-of-Thunder-NES-Unpublished" description="Days of Thunder was an unpublished NES game by developer Chris Oberth" image="https://images-na.ssl-images-amazon.com/images/I/61rhbiNhkoL._AC_.jpg" title="Days of Thunder"  %}
 
-### Infinity
```
Comment (RetroGameDeveloper) [inline pages/SourceCode/RetailConsoleSourceCode.md:40]: Why was Infinity removed?

File: pages/SourceCode/DecompiledRetailConsoleGames.md [hunk 2]
```diff
@@ -22,298 +22,245 @@ tags:
...
-
-### Super Mario Bros 2 The Lost Levels (Japanese)
-{% include link-to-other-site.html url="https://github.com/threecreepio/smb2j-disassembly" description="" image="https://upload.wikimedia.org/wikipedia/en/b/b5/Mariobros2japanbox.jpg" title="Disassembly of Super Mario Bros 2" %}
-
-### Super Mario Bros 3
-{% include link-to-other-site.html url="https://github.com/captainsouthbird/smb3" description="Looks like a fairly complete Disassembly of Super Mario Bros 3 that uses NESASM to re-assemble back into the original US ROM" image="https://upload.wikimedia.org/wikipedia/en/a/a5/Super_Mario_Bros._3_coverart.png" title="Disassembly of Super Mario Bros 3"  %}
-
-### Tetris
-{% include link-to-other-site.html url="https://github.com/CelestialAmber/TetrisNESDisasm" description="This is a disassembly of Tetris (NES) that uses cc65 as the assembler" image="https://upload.wikimedia.org/wikipedia/en/7/7d/Tetris_NES_cover_art.jpg" title="Disassembly of Tetris (NES)"  %}
+| Name | Project Type |
```
Comment (RetroGameDeveloper) [inline pages/SourceCode/DecompiledRetailConsoleGames.md:37]: This is awesome!

File: _config.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ logo: '/public/logo.png'
 highlighter: rouge
 
 # Site configurations
-gems:
+plugins:
```
Comment (RetroGameDeveloper) [inline _config.yml:11]: Not sure if this will work with Github pages version of Jekyll but we can give it a try

File: pages/ps1/PS1Symbols.md [hunk 4]
```diff
@@ -97,6 +97,7 @@ O.D.T. | Euro Demo 28 (France) | /ODT/MOVIES.SYM ODT.SYM
 SCARS | PlayStation Zone CD Vol. 8 (Germany) | SCARSPAL.SYM
 TH Pro Skater | Euro Demo 07-00 (Germany) | PROSKATE/MAIN.SYM
 UEFA EURO 2000 | Euro Demo 07-00 (Germany) | EURODEMO/BECODE_Z.SYM E2000_Z.SYM FECODE_Z.SYM
+Sled Storm | USA Demo Disc 24 | DEBUG/MAIN.MAP
```
Comment (RetroGameDeveloper) [inline pages/ps1/PS1Symbols.md:100]: Excellent!

File: categories/consoles/ps1.md [hunk 5]
```diff
@@ -79,8 +79,8 @@ Some classic PS1 games have been remastered or remade for more modern consoles,
 Name | Platform | Notes
 ---|---|---
 Ape Escape | PSP | Source Code was ported to the PSP as **Ape Escape: On the Loose**
-Crash Bandicoot 1-3 | PS4, Switch, PC, XboxOne | Remade in a new engine but tried to be as faithful as possible to the originals
-Crash Team Racing | PS4, Switch, PC, XboxOne | Remade in a new engine with additional content from Crash Nitro Cart and Crash tag Team Racing
+Crash Bandicoot 1-3 | PS4, Switch, PC, XboxOne | Remade in Alchemy engine but tried to be as faithful as possible to the originals
+Crash Team Racing | PS4, Switch, XboxOne | Remade in Alchemy Engine with additional content from Crash Nitro Kart and Crash Tag Team Racing
```
Comment (RetroGameDeveloper) [inline categories/consoles/ps1.md:83]: 👍

File: _config.yml (hunk 3 above)
Comment (FRDS) [reply _config.yml:11]: The term "gems" is now deprecated. GitHub Action when building Jekyll is the one actually warned me about it.
> Deprecation: The 'gems' configuration option has been renamed to 'plugins'. Please update your config file accordingly.

File: pages/SourceCode/RetailConsoleSourceCode.md (hunk 1 above)
Comment (FRDS) [reply pages/SourceCode/RetailConsoleSourceCode.md:40]: Linked repo was deleted or moved. I found [what might be where it moved](https://github.com/gb-archive/infinity-gbc), but I'm not 100% sure if it's the same content. Adding it back would be easy once it's confirmed.

**Review by RetroGameDeveloper — APPROVED** [review]

File: pages/SourceCode/DecompiledRetailConsoleGames.md (hunk 2 above)
Comment (FRDS) [reply pages/SourceCode/DecompiledRetailConsoleGames.md:37]: Thank you!

**Pull request merged**